app/assets/stylesheets/application.scss

```diff
@@ -16,6 +16,7 @@
 @import "charts/population_capacity";
 
 .tidy {
-  height: 600px;
-  overflow: scroll;
+  height: 400px;
+  margin-bottom: 3em;
```

Comment:
@jennifermarie I added a bottom margin to tidy so the _People who didn't post bond_ table wouldn't end at the bottom of the page.